docs/src/pages/docs/installation/kubernetes.mdx

```diff
@@ -94,25 +94,21 @@ postgresql:
 
 #### Dependencies
 
-You can specify pip packages to be installed before startup, e.g. to install extra database drivers:
+You can specify pip packages to be installed before startup. The following example installs the Big Query and Elasticsearch database drivers so that you can connect to those datasources in your Superset installation.
 
 ```yaml
-additionalRequirements:
-  - psycopg2
-  - redis
-  - elasticsearch-dbapi
-  - pymssql
-  - gsheetsdb
-  # Force verstion to work around https://github.com/betodealmeida/gsheets-db-api/issues/15
-  - moz-sql-parser==4.9.21002
-  # For OAuth
-  - Authlib
-  # For webdriver / reports
-  - gevent
+bootstrapScript: |
+  #!/bin/bash
+  apt-get update -y &&\
+  apt-get install -y --no-install-recommends nano &&\
+  rm -rf /var/lib/apt/lists/*
+  pip install psycopg2==2.8.5 \
```

Comment:
which versions are supported?

Reply:
these are, I think the important thing here is to get a working demo, it should be up  to the end-user to determine what packages they want to install in their own superset install.